sunpy/sun/models.py

```diff
@@ -84,3 +89,119 @@
 evolution = QTable(_t)
 evolution.source = 'Unknown'
 evolution.add_index('time')
+
+
+
+@u.quantity_input
+def diff_rot(duration: u.s, latitude: u.deg, rot_type='howard', frame_time='sidereal'):
```

Comment:
Is it time to rename this?

Reply:
what exactly, the function name? I don't recall renaming anything except ```_get_new_observation``` which I changed to ```_get_new_observation``` since it's no longer a private method.

Reply:
I did not say you renamed anything. This is a question for the other maintainers if we want to rename this while we are going through this process.

Reply:
Oh okay sorry I misunderstood.